Conflicting peer dependency in @next-auth/sequelize-adapter with next-auth@4.9.0

### Adapter type

@next-auth/sequelize-adapter

### Environment
```
  System:
    OS: macOS 11.5.2
    CPU: (8) x64 Intel(R) Core(TM) i7-1068NG7 CPU @ 2.30GHz
    Memory: 297.03 MB / 16.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 16.15.1 - ~/.nvm/versions/node/v16.15.1/bin/node
    Yarn: 1.22.5 - /usr/local/bin/yarn
    npm: 8.11.0 - ~/.nvm/versions/node/v16.15.1/bin/npm
    Watchman: 2022.05.16.00 - /usr/local/bin/watchman
  Browsers:
    Chrome: 103.0.5060.114
    Firefox: 102.0
    Safari: 14.1.2
  npmPackages:
    next-auth: 4.9.0 => 4.9.0


  npmPackages:
    @next-auth/sequelize-adapter: 1.0.4 => 1.0.4
```
### Reproduction URL

https://github.com/jbrunton/next-sequelize-adapter-test

### Describe the issue

The latest version of `@next-auth/sequelize-adapter` (1.0.4 at time of writing) has a conflicting peer dependency with the latest version of `next-auth` (4.9.0).

I was working around this by pinning `next-auth` to 4.5.0, but this version includes a security vulnerability which was patched in 4.9.0.

I'm not an expert with Node applications so it's possible I'm missing another workaround, but either way I imagine it should be possible to install the latest versions of these libraries without an error.

### How to reproduce

In an empty directory:

```
echo "{}" >> package.json
npm i next-auth @next-auth/sequelize-adapter
npm i
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: @next-auth/sequelize-adapter@1.0.4
npm ERR! Found: next-auth@4.9.0
npm ERR! node_modules/next-auth
npm ERR!   next-auth@"^4.9.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer next-auth@"4.5.0" from @next-auth/sequelize-adapter@1.0.4
npm ERR! node_modules/@next-auth/sequelize-adapter
npm ERR!   @next-auth/sequelize-adapter@"^1.0.4" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: next-auth@4.5.0
npm ERR! node_modules/next-auth
npm ERR!   peer next-auth@"4.5.0" from @next-auth/sequelize-adapter@1.0.4
npm ERR!   node_modules/@next-auth/sequelize-adapter
npm ERR!     @next-auth/sequelize-adapter@"^1.0.4" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

### Expected behavior

Installing the latest versions of next-auth and the sequelize-adapter should succeed.

--- a/packages/adapter-sequelize/package.json
+++ b/packages/adapter-sequelize/package.json
@@ -29,7 +29,7 @@
     "dist"
   ],
   "peerDependencies": {
-    "next-auth": "workspace:*",
+    "next-auth": "^4",
     "sequelize": "^6.6.5"
   },
   "devDependencies": {

